mandatory/3-function-output.js

```diff
@@ -1,14 +1,17 @@
 // Add comments to explain what this function does. You're meant to use Google!
+// This function generates a random number between 0 and 10
```

Comment:
Thought: This is correct, but it's worth noting that the number may not be an integer, and could be 0 but cannot be 10.
See https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math/random for more info